docs/variants/novacustom_nv4x_adl/heads.md

```diff
@@ -0,0 +1,205 @@
+# Dasharo (coreboot + Heads) firmware variant
+
+For the NovaCustom NV4x 12th Gen, Heads-based variant of Dasharo firmware is
+offered as a Technology Preview Release.
+
+Please consider supporting the project financially by purchasing the
+[Dasharo Entry Subscription](https://novacustom.com/product/dasharo-entry-subscription/).
+With this subscription, you get access to the
+[Transition from Dasharo UEFI to Heads variant](#transition-from-dasharo-uefi-to-heads-variant)
+feature in Dasharo Tools Suite and support from Dasharo directly via Matrix.
+The subscription is not automatically renewed.
+
+Following Release Notes describe status of development of Dasharo (coreboot +
+Heads) firmware for NovaCustom NV4x 12th Gen.
+
+## v0.9.0 - 2024-02-28
+
+### Changed
+
+- [This is a Dasharo Entry Subscription release](https://docs.dasharo.com/dev-proc/versioning/#dasharo-entry-subscription-releases)
+- Heads Linux is used as a payload
+
+### Known issues
+
+- [Power button does not work in Qubes](https://github.com/Dasharo/dasharo-issues/issues/710)
+- [Heads shuts down instead of rebooting](https://github.com/Dasharo/dasharo-issues/issues/711)
+- [Existing Qubes installation is not found as bootable after transition back to EDK2](https://github.com/Dasharo/dasharo-issues/issues/713)
+- [Builds are not fully reproducible](https://github.com/linuxboot/heads/issues/1616)
+
+### Binaries
+
+[sha256][novacustom_nv4x_adl_v0.9.0_heads.rom_hash]{.md-button}
+[sha256.sig][novacustom_nv4x_adl_v0.9.0_heads.rom_sig]{.md-button}
+
+This is a Dasharo Entry Subscription Release. To obtain access to the pre-built
+binaries you will have to
+[become the Dasharo Entry Subscription subscriber](../../ways-you-can-help-us.md#become-a-dasharo-entry-subscription-subscriber).
+You will get the access to all of the firmware updates for the duration of the
+subscription via Dasharo Entry Subscription newsletter.
+
+To verify binary integrity with hash and signature please follow the
+instructions in [Dasharo release signature verification](/guides/signature-verification)
+using [this key](https://github.com/3mdeb/3mdeb-secpack/blob/master/customer-keys/novacustom/dasharo-release-0.9.x-for-novacustom-signing-key.asc)
+
+### SBOM (Software Bill of Materials)
+
+- [Dasharo heads fork based on v0.2.0 revision ccf49703](https://github.com/Dasharo/heads/tree/ccf49703)
+- [Dasharo coreboot fork based on 4.21 revision 3a9aa3a4](https://github.com/Dasharo/edk2/tree/3a9aa3a4)
```

Comment:
```suggestion
- [Dasharo coreboot fork based on 4.21 revision 3a9aa3a4](https://github.com/Dasharo/coreboot/tree/3a9aa3a4)
```

I guess an error in the scripts? @miczyg1

Reply:
fixed by fd6d63664d35

Reply:
Yup, one sec